### ISSUE TITLE
How to correctly display the original data?

### ISSUE THREAD

**Issue by Stas384** [zongyi-li/fourier_neural_operator#83]
Interested in the question of solving the Burger equation in the case of the dependence of the solution on time and coordinates. However, when studying the file embedded on Google Drive, I still did not fully understand what a, a_x, and their smoothed versions mean. With u, everything is clear, this is the solution to the equation. And also, do I understand correctly that it is enough to train the network with initial constraints from a certain range, and it will be able to issue a solution with any specific initial conditions from this range without any problems? The following question arises, how does the network understand which equation it solves? Or can it be called an almost date driven approach?

**Comment by zongyi-li** [comment]
Sorry for the delay. In the case of Burgers, the input **a** is just the initial conditions. **a_x** is the derivatives da/dx (not used in the code.)  The setup is data-driven. It does not know about the underlying equation. If the equation is available it's possible to use the physics-inform learning https://github.com/devzhk/PINO.

**Comment by Stas384** [comment]
@zongyi-li  It turns out that in order to train a neural network to solve partial differential equations with different initial approximations, some array of a pre-solved equation is required, right? And even if there is an equation, is this array still needed? Is it possible to teach a neural network to solve a differential equation under various initial conditions without having an array of solutions, but having only the form of an equation?

**Comment by zongyi-li** [comment]
Yes, if the equation is known, we don't need an array of pre-solved solutions. We just need to sample random initial conditions (input) and apply the equation loss on each of them.  Please check the paper https://arxiv.org/abs/2111.03794. The code is available at https://github.com/devzhk/PINO.
